tests/test_regression.py

```diff
@@ -485,6 +485,17 @@ def test_30_remove(self):
 
         expected = ['5.150.145.0/24', '109.161.64.0/20']
         self.assertEqual(expected, [n.prefix for n in tree])
+		
+	def test_31_covering_with_delete_error(self):
```

Comment:
This seems to be tabbed in instead of 4 spaces leading to a syntax error during tests.